app/browser/api/ledger.js

```diff
@@ -1117,6 +1119,38 @@ const pageDataChanged = (state, viewData = {}, keepInfo = false) => {
   return state
 }
 
+const checkBrowserActivityTime = (state) => {
+  // Fuzzing threshold
+  const minTime = 30 * ledgerUtil.milliseconds.minute
+  const publishers = ledgerState.getPublishers(state)
+
+  if (publishers.isEmpty()) {
+    return state
+  }
+
+  const publisherKeys = Array.from(publishers.keys())
```

Comment:
This one can be removed and do a reducer directly on `publishers` list.

Reply:
`publishers` is a `Map`, `reduce` should only work on Arrays or other iterables, can we leave this one?

Reply:
done